digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectSectionMValueCalculator.scala

```diff
@@ -51,8 +51,7 @@ object ProjectSectionMValueCalculator {
       val addressValue = if (someCalibrationPoint.nonEmpty) someCalibrationPoint.get.addressMValue else m + pl.geometryLength * coEff
       pl.status match {
         case LinkStatus.New => addressValue
-        case LinkStatus.Transfer | LinkStatus.NotHandled | LinkStatus.Terminated | LinkStatus.Numbering => m + pl.addrMLength
-        case LinkStatus.UnChanged => pl.endAddrMValue
+        case LinkStatus.Transfer | LinkStatus.NotHandled | LinkStatus.Terminated | LinkStatus.Numbering | LinkStatus.UnChanged => m + pl.addrMLength
```

Comment:
VIITE-2062 has also change for this place. Terminated would be like UnChanged was here before.

Reply:
yep i noticed that also